1_space_odyssey/js/mannuel-9000/peder-9000/app.js

```diff
@@ -136,3 +136,63 @@ if (hasJetPack) {
     "JetPack NOT enabled, please enable jetPack before doing a space walk"
   );
 }
+
+/*
+ 3. METEOR SHOWER::::::
+    On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+    a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+      create an if condition in your loop to print only 1 response using a comparison operator.
+*/
+console.log("\u{1F327} METEOR SHOWER::::::");
+
+var responses = [
+  "Call basecamp and ask for assistance, but they are far away",
+  "Do nothing, just push through",
+  "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+  "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+];
+
+// write your code here, use the console log beneath
+
+for (var count = 0; i <= responses.length; count++) {
```

Comment:
noice it works.
standardise your counter variable name. use `i` everywhere, or `count` every where